app/renderer/js/main.js

```diff
@@ -621,6 +630,20 @@ class ServerManagerView {
 			});
 		}
 
+		ipcRenderer.on('language-change', (event, { serverLanguage, domain }) => {
+			const index = this.getTabIndex(domain);
+			const oldLanguage = this.tabs[index].props.language;
+			this.tabs[index].props.language = serverLanguage;
+			if (oldLanguage !== serverLanguage) {
```

Comment:
This is to handle the case where both org have the same language, right?

Reply:
Uh, yeah, pretty much. More accurately, this is for the case when the app default (English) is not the same as the server language.